### PR TITLE
fix: footer css pollutes the whole

### PR DESCRIPTION
**Pull request by QC-L**
I found that the footer of the project has disappeared.

### CODE REVIEW

File: website/core/Footer.js [hunk 1]
```diff
@@ -13,56 +13,59 @@ class Footer extends React.Component {
...
+              <a href={this.props.config.baseUrl + "blog"}>Blog</a>
+              <a href="https://github.com/babel">GitHub Org</a>
+              <a href="https://github.com/babel/babel">GitHub Repo</a>
+              <a href="https://github.com/babel/website">Website Repo</a>
+              <a href="https://old.babeljs.io">Old 6.x Site</a>
+              <a href="http://henryzoo.com/babel.github.io">Old 5.x Site</a>
+            </div>
+          </section>
+        </footer>
+        <script src="/scripts/hide-footer.js" />
```
Comment (existentialism) [inline website/core/Footer.js]: Let's move this to https://github.com/babel/website/blob/master/website/siteConfig.js#L126?

Comment (QC-L) [reply website/core/Footer.js]: OK, I move it.

**Comment by existentialism** [comment]
@QC-L thanks!

**Comment by QC-L** [comment]
@existentialism Not at all.

**Pull request merged**